docs/Set-PSReadLineOption.md

```diff
@@ -500,7 +500,7 @@ Aliases:
 
 Required: False
 Position: Named
-Default value: On Windows - ~\AppData\Roaming\Microsoft\Windows\PowerShell\PSReadLine\$($host.Name)_history.txt otherwise $XDG_DATA_HOME/powershell/PSReadLine/$($host.Name)_history.txt or $HOME/.local/share//powershell/PSReadLine/$($host.Name)_history.txt
+Default value: On Windows - $env:APPDATA\Microsoft\Windows\PowerShell\PSReadLine\$($host.Name)_history.txt otherwise $XDG_DATA_HOME/powershell/PSReadLine/$($host.Name)_history.txt or $HOME/.local/share/powershell/PSReadLine/$($host.Name)_history.txt
```

Comment:
I suppose you should use PowerShell syntax for the non-Windows use of environment variables too.

Reply:
I wanted to, but I couldn't find an environment variable on Linux that points to $HOME/.local/share.

Reply:
And I wonder if it would be more readable if `$($host.Name)_history.txt` was not repeated 3 times, e.g.

> Default value: A file named $($host.Name)_history.txt in $env:APPDATA\Microsoft\Windows\PowerShell\PSReadLine on Windows otherwise $env:XDG_DATA_HOME/powershell/PSReadLine or $env:HOME/.local/share/powershell/PSReadLine

Reply:
Is $HOME not the environment variable?

Reply:
It is - verified checking the code.

Reply:
It is and you use it already.

Reply:
ah, I got it.

Reply:
Linux has $HOME and $env:HOME. Is it the same on macOS for $env:XDG_DATA_HOME and $XDG_DATA_HOME?